setup.cfg

```
@@ -63,7 +63,8 @@ setup_requires =
 install_requires =
     backports.functools_lru_cache
     six>=1.11.0
-    more_itertools>=2.6
+    more_itertools>=2.6<6; python_version<'3'
```

The comma is missing
```suggestion
    more_itertools>=2.6,<6; python_version<'3'
```